runtime/oti/j9nonbuilder.h

```diff
@@ -4875,6 +4875,21 @@ typedef struct J9InvocationJavaVM {
 	J9NativeLibrary * reserved2_library;
 } J9InvocationJavaVM;
 
+#if JAVA_SPEC_VERSION >= 19
+typedef struct J9VMContinuation {
+	UDATA* arg0EA;
+	UDATA* bytecodes;
+	UDATA* sp;
+	U_8* pc;
+	struct J9Method* literals;
+	UDATA* stackOverflowMark;
+	UDATA* stackOverflowMark2;
+	J9JavaStack* stackObject;
+	struct J9VMContinuation* linkNext;
+	struct J9VMContinuation* linkPrevious;
+} J9VMContinuation;
+#endif /* JAVA_SPEC_VERSION >= 19 */
+
```

Comment:
Since GC will fully maintain Continuation Lists(list of Continuation Objects instead of J9VMContinuation), we don't need to linkNext and linkPrevious in J9VMContinuation(there would hidden field continuationLink in Continuation Oject for the list) and also it is hard to maintain linkNext and linkPrevious from GC, we can provide an iterator for retrieving the list of J9VMContinuation.

Reply:
The VM needs those fields

Reply:
but GC will not maintain(set and update) both  linkNext and linkPrevious during collections.

Reply:
GC will call a vmHook for that